app/src/main/java/com/sun/makeupwindow/ui/search/SearchContract.kt

```diff
@@ -0,0 +1,16 @@
+package com.sun.makeupwindow.ui.search
+
+import com.sun.makeupwindow.base.BasePresenter
+import com.sun.makeupwindow.base.BaseView
+import com.sun.makeupwindow.data.model.Product
+
+interface SearchContract {
+    interface View : BaseView {
+        fun showProducts(product: List<Product>)
+    }
+
+    interface Presenter : BasePresenter {
+        fun getProducts(wordSearch: String)
```

Comment:
```suggestion
        fun getProducts(searchWord: String)
```
không như a suggest trc thì tn hợp lý hơn